src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -95,15 +101,18 @@ private static void LoadZipFile(Dictionary<string, Lazy<ExternalXrefSpec>> resul
                     var xrefMap = JsonUtility.Deserialize<XrefMapModel>(errors, reader, path);
                     foreach (var spec in xrefMap.References)
                     {
+                        spec.RepoUrl = xrefMap.RepositoryUrl;
                         result.TryAdd(spec.Uid, new Lazy<ExternalXrefSpec>(() => spec));
                     }
                 }
             }
         }
 
-        private static List<(string uid, long start, long end)> GetXrefSpecPositions(ReadOnlySpan<byte> content)
+        private static (List<(string uid, long start, long end)>, Dictionary<string, string>) GetXrefSpecPosAndStrProperties(
```

Comment:
consider return the `repositoryUrl` directly here.
```suggestion
        private static (List<(string uid, long start, long end)>, string repositoryUrl) GetXrefSpecPosAndStrProperties(
```

Reply:
OK